presto-docker-image/Dockerfile

```diff
@@ -21,40 +21,18 @@ RUN \
     yum -y -q install java-11-openjdk-devel less && \
     yum -q clean all && \
     rm -rf /var/cache/yum && \
-    rm -rf /tmp/* /var/tmp/*
-
-RUN \
-    set -xeu && \
-    mkdir -p /data/presto
+    rm -rf /tmp/* /var/tmp/* && \
+    groupadd presto --gid 1000 && \
+    useradd presto --uid 1000 --gid 1000 && \
+    mkdir -p /usr/lib/presto && \
+    mkdir -p /data/presto && \
```

Comment:
`mkdir` takes multiple paths, so you could create these together, to be symmetric with the `chown`

Reply:
👍